src/components/Card/Card.stories.tsx

```diff
@@ -16,9 +16,12 @@ stories.add(
         <Card>
           <CardHeader subtitle='Do you Know Lorem Ipsum?'>Lorem Ipsum</CardHeader>
           <CardBody>
+            <Text>
             Lorem Ipsum is simply dummy text of the printing and
-            typesetting industry. Lorem Ipsum has been the industry's
-            standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+            typesetting industry. <Text inline>Lorem Ipsum has been the industry's
+            standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+            scrambled it to make a type specimen book.</Text>
+            </Text>
```

Comment:
Card should use `Text` component internally.

Reply:
Doesn't know how to fix this. Meanwhile, I'm removing the usage of Text inside Card Component.